react/features/conference/components/native/styles.js

```diff
@@ -32,6 +34,53 @@ export default createStyleSheet({
         fontSize: 14
     },
 
+    gradient: {
```

Comment:
(I'm comenting her,e but this is a general coment about the style)

The gradient is way to tall and there is too big of a gap between the top of the screen and the content, it feels like it floats. (see screenshot).

I'd imagine this would as tall as the header we have on the welcome page, and the text shows with the same amount of margin.

![img_0026](https://user-images.githubusercontent.com/317464/52116007-5d68eb00-2610-11e9-8136-6091e50db88d.PNG)

Reply:
Please dicuss it with Yana and/or Tibi to change the design, it's not my call. I like it this way though, and don't forget that we need to take care of the notch too

Reply:
This design doesn't match the spec. The spec also has some mistakes, but currently this implementation doesn't align with it either.